### DISCUSSION TITLE
Send a break command.

### DISCUSSION THREAD

**Discussion by licketylips** [Eugeny/tabby#8077] [General]
Is there a menu item to send a break command in terminal console mode? My laptop keyboard does not have a break key. If not is this functionality able to be added?

(this function exists in Putty / Tera Term)

**Comment by MarioMan632** [comment]
I love Tabby and want to use it for my job but I do sometimes need to insert a break which can be done with Alt+B or a special command in Tera Term/Putty and it would make this perfect if this was added to Tabby.
